flight/PiOS/STM32/inc/pios_sbus_priv.h

```diff
@@ -85,6 +86,8 @@ struct pios_sbus_cfg {
 	BitAction gpio_inv_disable;
 };
 
+#endif /* PIOS_INCLUDE_USART */
```

Comment:
I don't see a matching ifdef for this? Looks like it will terminate the header guard one.